tests/unit/slices/filters.ts

```diff
@@ -1,65 +1,130 @@
+    filter,
     selected: false,
     displayName: 'some label'
   };
 
+  const disjunctionFilter: StaticFilter = {
+    kind: 'disjunction',
+    combinator: FilterCombinator.OR,
+    filters: [{
+      kind: 'fieldValue',
+      fieldId: 'c_someField',
+      matcher: Matcher.Equals,
+      value: 'a value'
+    }, {
+      kind: 'fieldValue',
+      fieldId: 'c_anotherField',
+      matcher: Matcher.Equals,
+      value: 'separate value'
+    }]
+  };
+  const selectedDisjunctionFilter: SelectableStaticFilter = {
+    filter: disjunctionFilter,
+    selected: true,
+    displayName: 'different label'
+  };
+
+  const conjunctionFilter: StaticFilter = {
+    kind: 'conjunction',
+    combinator: FilterCombinator.AND,
+    filters: [{
+      kind: 'fieldValue',
+      fieldId: 'c_someField',
+      matcher: Matcher.Equals,
+      value: 'different value'
+    }, {
+      kind: 'conjunction',
+      combinator: FilterCombinator.AND,
+      filters: [{
+        kind: 'fieldValue',
+        fieldId: 'c_anotherField',
+        matcher: Matcher.Equals,
+        value: 'another value'
+      }, {
+        kind: 'fieldValue',
+        fieldId: 'c_uniqueField',
+        matcher: Matcher.Equals,
+        value: 'unique value'
+      }]
+    }]
+  };
+  const selectedConjunctionFilter: SelectableStaticFilter = {
+    filter: conjunctionFilter,
+    selected: true,
+    displayName: 'other display name'
+  };
+
   const initialState: FiltersState = {
     static: [
       {
-        fieldId: 'id1',
-        matcher: Matcher.Equals,
-        value: 'value1',
+        filter: {
+          kind: 'fieldValue',
+          fieldId: 'id1',
+          matcher: Matcher.Equals,
+          value: 'value1'
+        },
         selected: true,
         displayName: 'label1'
       },
       {
-        fieldId: 'id2',
-        matcher: Matcher.Equals,
-        value: 'value2',
+        filter: {
+          kind: 'fieldValue',
+          fieldId: 'id2',
+          matcher: Matcher.Equals,
+          value: 'value2'
+        },
         selected: true,
         displayName: 'label2'
       },
       {
-        fieldId: 'id3',
-        matcher: Matcher.Equals,
-        value: 'value3',
+        filter: {
+          kind: 'fieldValue',
+          fieldId: 'id3',
+          matcher: Matcher.Equals,
+          value: 'value3'
+        },
         selected: false,
         displayName: 'label3'
-      }
+      },
+      selectedDisjunctionFilter
     ]
   };
 
   it('setStatic action is handled properly', () => {
-    const staticFilter = [selectableFilter];
-    const actualState = reducer({}, setStatic(staticFilter));
+    const staticFilters = [selectableFilter, selectedDisjunctionFilter, selectedConjunctionFilter];
+    const actualState = reducer({}, setStatic(staticFilters));
     const expectedState = {
-      static: staticFilter
+      static: staticFilters
     };
 
     expect(actualState).toEqual(expectedState);
```

Comment:
Think this is more personal preference but what do you think of `expect(actualState).toEqual({ static: staticFilters })`?

Reply:
it does seem a little unnecessary for simple objects like these, but I like that the `expectedState` variables are consistent with our other tests where there are more complex objects, which is why I left them in. but, if you have a strong preference, I can remove them for the simpler cases

Reply:
We can keep them! Was wondering what we prefer as a team